tools/makem.js

```diff
@@ -174,6 +174,7 @@ DEBUG_FLAGS += ' -gsource-map -fsanitize=undefined ';
 DEBUG_FLAGS += ' -s ASSERTIONS=2 '
 DEBUG_FLAGS += ' --profiling '
 DEBUG_FLAGS += '  -s DEMANGLE_SUPPORT=1 ';
+DEBUG_FLAGS += ' -DWEBARKIT_DEBUG '
```

Comment:
This will enable WEBARKIT_LOGd in debug mode. But it is required also to set the `webarkitLogLevel` to the debug mode. In order to do this is it is needed to create a setter in the C++ code to set `webarkitLogLevel` to **WEBARKIT_LOG_LEVEL_DEBUG**.

Reply:
The `setLogLevel` method within set the `webarkitLogLevel` should be implemented in a Manager class where instantiate a new **WebARKitTracker** object and other utilities functions.

Reply:
this was done with https://github.com/webarkit/WebARKitLib/pull/18/commits/df0493accfa64066a4ccd078901ff0d2e74faea5